.github/workflows/build.yml

```diff
@@ -23,24 +23,43 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
           cache: yarn
-      - run: yarn
+      - run: yarn install --ignore-scripts
+      - run: yarn build:cjs
       - run: tar -czf /tmp/web3-${{ matrix.node }}.js.tar.gz --exclude="./.git" ./
       - uses: actions/upload-artifact@v3
         with:
           name: web3-${{ matrix.node }}.js.tar.gz
           path: /tmp/web3-${{ matrix.node }}.js.tar.gz
-
+  build-cjs-types:
+      name: Build ESM & Types
```

Comment:
There was a ESM tests PR by @luu-alex , are these running after this build?

Reply:
doesnt seem

Reply:
change node version in esm tests.